ocs_ci/framework/main.py

```diff
@@ -66,6 +66,15 @@ def init_ocsci_conf(arguments=None):
     )
     parser.add_argument("--ocs-registry-image")
     parser.add_argument("--flexy-env-file", default="", help="Path to flexy env file")
+    parser.add_argument(
+        "--disable-components",
+        nargs="+",
```

Comment:
If I remember it correctly, the `nargs="+"` means, that you can use multiple values for one option (space separated) - e.g.:
```
--disable-components rgw cephfs blockpool
```
I would not recommend this type of handling as it might be slightly unclear in our case (what is the parameter value and what is next parameter).

Reply:
> @clacroix12 @vavuthu @dahorak @vasukulkarni ,
> With this PR we are introducing a new cli option `--disable-components` followed by component names to be disabled from deployment. Currently this PR implements a comma separated component names like
> 
> ```python
> --disable-components rgw,cephfs,blockpools
> ```
> 
> However @petr-balogh suggested we go with
> 
> ```python
> --disable-component rgw --disable-components cephfs --disable-component blockpool
> ```
> 
> which one you guys would feel more intuitive and better approach

We have other CLI options in which we use the append action (what petr suggested). I think we should be consistent and not introduce different means of adding multiple values to CLI options. One [example](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/main.py#L61) of this is `--ocsci-conf`. Because we configure it to use the append action, we can call `--ocsci-conf` multiple times and the values get added to a list automatically.

Reply:
The consistency with `--ocsci-conf` is good point and I fully support this option - that's really good point. That means, that you can repeat the whole combination of parameter name and value - e.g. `--ocsci-conf config1.yaml --ocsci-conf config2.yaml`, which is enabled via `action="append"` parameter in `parser.add_argument(...)` ([python doc](https://docs.python.org/3/library/argparse.html#action), [usage in ocs-ci](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/main.py#L61)).
But my point in the comment above was against using `nargs="+"`, which will allow you to put multiple values to single parameter like this: `--ocsci-conf config1.yaml config2.yaml` ([python doc](https://docs.python.org/3/library/argparse.html#nargs). This syntax is might be slightly confusing to distinguish what is positional parameter and what is value for named parameter, so if it is not necessary, I would avoid it.

Reply:
@dahorak I totally agree with you. I must have replied to your comment instead of making a new one myself.